jsonparse.js

```diff
@@ -51,9 +51,10 @@ var TAB =             "\t".charCodeAt(0);
 
 var STRING_BUFFER_SIZE = 64 * 1024;
 
-function Parser() {
+function Parser(options = {}) {
```

Comment:
We should make sure we're using a node version that supports defaults.

Reply:
Could we include the options in the default values? Something like this:
```suggestion
function Parser({parseNumbersAsString = false} = {}) {
```